docs/_documentation/fundamentals/value-combiners.md

```diff
@@ -11,7 +11,7 @@ Tibet binding (see [wiki/Databinding](https://www.mvvmcross.com/documentation/fu
 
          local:MvxBind="Text Format('{0} {1} {2}', Greeting(Gender), FirstName, LastName)"
          
-The rules and mechanisms for registering ValueCombiners are similar to those for registering ValueConverters. However, because combiners are not commonly declared in user code, MvvmCross doesn't current perform a Reflection sweep across your Core or UI assemblies. If you do want to add is used by default on the , then a ValueCombiner class named `FooValueCombiner` will be registered under the name `Foo`.
+The rules and mechanisms for registering ValueCombiners are similar to those for registering ValueConverters. However, because combiners are not commonly declared in user code, MvvmCross doesn't current perform a Reflection sweep across your Core or UI assemblies. If you do want to add by default, then a ValueCombiner class named `FooValueCombiner` will be registered under the name `Foo`.
```

Comment:
I think I found a typo

MvvmCross doesn't `current` perform

should be

MvvmCross doesn't currently perform

Reply:
Yeah, thank for your addition :D